### DISCUSSION TITLE
dotnet new

### DISCUSSION THREAD

**Discussion by jordi-z80** [kniEngine/kni#1751] [Q&A]
Hi! 

Is it possible to create KNI projects via commandline with dotnet new?

Thanks!

**Comment by jordi-z80** [comment]
Never mind, I just read the discussion below https://github.com/kniEngine/kni/discussions/687 so I suppose it can't be done yet.

**Comment by vargasbarbarad** [comment]
![IMG_0442 JPG](https://github.com/user-attachments/assets/92b80711-b5bc-4d26-94ca-a67cdd2b987a)
